### PR TITLE
Enhancement/skale 2713 update to sm 1.40

### PR DESCRIPTION
**Pull request by dmitry-tk**

### CODE REVIEW

File: skale/utils/helper.py [hunk 1]
```diff
@@ -35,7 +35,7 @@ def decapitalize(s):
     return s[:1].lower() + s[1:] if s else ''
 
 
-def format_fields(fields):
+def format_fields(fields, flist=False):
```
Comment (badrogger) [inline skale/utils/helper.py:38]: I think it's better to split this logic into different decorators

File: skale/contracts/schains.py [hunk 2]
```diff
@@ -0,0 +1,91 @@
...
+
+FIELDS = [
+    'name', 'owner', 'indexInOwnerList', 'partOfNode', 'lifetime', 'startDate', 'startBlock',
+    'deposit', 'index', 'chainId'
+]
+
+
+class SChains(BaseContract):
+
+    def get_schains_internal(self):
```
Comment (badrogger) [inline skale/contracts/schains.py:35]: I think we should use cached property here

File: skale/contracts/dkg.py [hunk 3]
```diff
@@ -20,6 +20,21 @@
 from skale.contracts import BaseContract, transaction_method
 from skale.transactions.tools import retry_tx
 from skale.utils.constants import GAS
+from skale.utils.helper import split_public_key
+
+
+class KeyShare():
```
Comment (badrogger) [inline skale/contracts/dkg.py:26]: I think it's better to remove braces in the end. It doesn't do anything except two extra symbols.

File: skale/contracts/schains.py (hunk 2 above)
Comment (dmitry-tk) [reply skale/contracts/schains.py:35]: done

File: skale/contracts/dkg.py (hunk 3 above)
Comment (dmitry-tk) [reply skale/contracts/dkg.py:26]: +

**Pull request merged**